resources/views/hr/application/index.blade.php

```diff
@@ -46,167 +45,157 @@
+                    <h5 class="closed">Closed</h5>
                 </a>
             </li>
-        </ul>
-        @if( isset($openJobsCount, $openApplicationsCount) )
-        <div class="alert alert-info mb-2 p-2">
-            <span>There are <b>{{ $openJobsCount }}</b> open jobs and <b>{{ $newApplicationsCount }}</b> open
-                applications</span>
         </div>
-        @endif
+    </div>    
+    @if( isset($openJobsCount, $openApplicationsCount) )
+    <div class="alert alert-info mb-2 p-2">
+        <span>There are <b>{{ $openJobsCount }}</b> open jobs and <b>{{ $newApplicationsCount }}</b> open
+            applications</span>
+    </div>
+    @endif
+    <div class="table">
+        <table class="table table-striped" id="applicants_table">
+            <thead>
+                <th class="name">Name</th>
+                <th class="details">Details</th>
+                <th>
+                    <span class="dropdown-toggle c-pointer assignee" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="assigneeDropdown">Assignee</span>
+                    <div class="dropdown-menu" aria-labelledby="assigneeDropdown">
+                        <span class="dropdown-item-text fz-12">Filter by assignee</span>
+                        @foreach ($assignees as $assignee)
+                            @php
+                                $target = route(request()->route()->getName(), ['assignee' => [$assignee->id]]);
+                                $class = in_array($assignee->id, request()->get('assignee') ?? []) ? 'visible' : 'invisible';
+                            @endphp
+                            <a class="dropdown-item" href="{{ $target }}">
+                                <i class="fa fa-check fz-12 {{ $class }}"></i>
+                                <img src="{{ $assignee->avatar }}" alt="{{ $assignee->name }}"
+                                    class="w-20 h-20 rounded-circle mr-1">
+                                <span>{{ $assignee->name }}</span>
+                            </a>
+                        @endforeach
+                    </div>
+                </th>
+                <th>
+                    <span class="dropdown-toggle c-pointer status" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="statusDropdown">Status</span>
+                    <div class="dropdown-menu" aria-labelledby="statusDropdown">
+                        <span class="dropdown-item-text fz-12">Filter by status</span>
+                        @foreach ($tags as $tag)
+                            @php
+                                $target = request()->fullUrlWithQuery(['tags' => [
+                                $tag->id
+                                ]]);
```

Comment:
It is good pratice to use `array()` instead of `[ ]`
```suggestion
                                $target = request()->fullUrlWithQuery(array(
                                    'tags' => array($tag->id)
                                 ));
                
```

Reply:
@GauravGusain98 I have not changed these lines. It was pre written.